### DISCUSSION TITLE
Project passed :) - What is next?

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#242] [Project: Privacy Product Specification] [answered]
Cool!
Thank you so much for taking the time to correct the final project!
I bet it was a lot of work to get all the reviewers aligned and trained.

I am looking forward to the next more technical bit of the course. There was a big delay on the correction, I assume there will also be a big one for the next part: Foundations of private computation. Do you know an optimistic and pessimistic starting date?

Thank you so much again for all the effort!

Best regards,
Gonzalo

**Comment by yemikifouly** [comment, marked as answer]
Hello Gonzalo! 👋 

Yay!!! Congratulations on passing the project and completing the course. 🎉

Onto your questions 🙂:

* Yes. There was indeed a big delay in the review of the projects for the first course **but** we are working to ensure that this doesn't happen for the upcoming course. Ideally, our goal is to have students receive feedback for their submissions in a matter of hours and this is what we are now working to make happen. 🙂 
* For the second course, I don't have a date. I know that the content creators are hard at work but no date has been shared yet. I expect one will be announced in the next couple of weeks or so.

I am really glad you enjoyed the first course and are now looking forward to the next course Gonzalo! 🙂
